core/backend/package.json

```diff
@@ -68,7 +68,7 @@
     "@types/fs-extra": "^4.0.7",
     "@types/mocha": "^8.2.2",
     "@types/multiparty": "^0.0.31",
-    "@types/node": "16.11.7",
+    "@types/node": "18.7.18",
```

Comment:
do we want to be pinning it to an exact version? or should we be using ^ to pull the latest types?

Reply:
> do we want to be pinning it to an exact version? or should we be using ^ to pull the latest types?

Wouldn't this silently pull us up to node versions that we aren't officially supporting yet, though?

Reply:
Using ^ should be fine, itll only pull the latest minor/patch in this major, and Node follows semver better than us, so we can expect no breaking changes

Reply:
Done.